src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/DescriptorReferenceValidation.feature.disabled

```diff
@@ -0,0 +1,89 @@
+                  """
+                  {
+                        "localEducationAgencyId": 25590100,
+                        "nameOfInstitution": "Grand Institution Test",
+                        "categories": [
+                        {
+                        "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor"
+                        }
+                        ],
+                        "localEducationAgencyCategoryDescriptor": "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor"
+                  }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                  {"detail":"Data validation failed. See 'validationErrors' for details.","type":"urn:ed-fi:api:bad-request:data","title":"Data Validation Failed","status":400,"correlationId":null,"validationErrors":{"$.localEducationAgencyCategoryDescriptor":["LocalEducationAgencyCategoryDescriptor value 'uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Fake' does not exist."],"$.categories[0].educationOrganizationCategoryDescriptor":["EducationOrganizationCategoryDescriptor value 'uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Fake' does not exist."]}}
+                  """
+
+        @ignore
+        Scenario: User can POST a resource when descriptor exists
+             When client sends a POST request to "/ed-fi/localEducationAgencies" with body
+                  """
+                  {
+                    "localEducationAgencyId": 25590100,
+                    "nameOfInstitution": "Grand Institution Test",
+                    "categories": [
+                        {
+                            "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Education+Service+Center"
+                        }
+                    ],
+                    "localEducationAgencyCategoryDescriptor": "uri://ed-fi.org/LocalEducationAgencyCategoryDescriptor#Public+school+district+part+of+a+supervisory+union"
+                  }
+                  """
+             Then it should respond with 201
+
+
+        @ignore
+        Scenario: User can UPDATE a resource when descriptor exists
+             When client sends a PUT request to "/ed-fi/students/{id}" with body
+                  """
+                  {
+                    "studentUniqueId": "604824",
+                    "birthDate": "2010-01-13",
+                    "firstName": "Traci",
+                    "lastSurname": "Mathews",
+                    "citizenshipStatusDescriptor": "uri://ed-fi.org/CitizenshipStatusDescriptor#US+Citizen"
+                  }
+                  """
+             Then it should respond with 204
+
+
+        @ignore
+        Scenario: User CANNOT update a resource using a descriptor that does not exist
+             When client sends a POST request to "/ed-fi/students/{id}" with body
```

Comment:
A POST request cannot have an ID in the URL: only GET, PUT, and DELETE requests can have a specific identifier. This is a standard part of REST API design, not just Ed-Fi.

An Ed-Fi API _can_ update an existing record on POST, if the natural key matches a record that is already in the database. In this case the natural key is `studentUniqueId`, so if you change the value below to `604824` (matching the Background setup), then this POST request will be an _update to an existing record_.